### ISSUE TITLE
GUI Crash loading a RADV-exported profile

### ISSUE THREAD

**Issue by farnoy** [GPUOpen-Tools/radeon_gpu_profiler#77]
I'm using RGP 1.14 on Windows 11 to open an .rgp trace exported by radv. The GUI crashes for me while opening the trace file, there's no stdout/stderr message nor a crash dump.

I tried debugging under WinDbg and the stack trace for the crash seems to be:

```
[0x0]   RadeonGPUProfiler + 0x5ad900   
[0x1]   Qt5Core!QAbstractItemModel::hasChildren + 0x16   
[0x2]   Qt5Core!QSortFilterProxyModel::hasChildren + 0x5f   
[0x3]   Qt5Widgets!QTreeViewPrivate::hasVisibleChildren + 0x46   
[0x4]   Qt5Widgets!QTreeViewPrivate::layout + 0x788   
[0x5]   Qt5Widgets!QTreeView::doItemsLayout + 0x3cd   
[0x6]   Qt5Widgets!QTreeView::resizeColumnToContents + 0x3f   
[0x7]   RadeonGPUProfiler + 0x443984   
```

The trace I captured from `vkcube`: [vkcube_2022.12.29_16.16.56.zip](https://github.com/GPUOpen-Tools/radeon_gpu_profiler/files/10320259/vkcube_2022.12.29_16.16.56.zip)

**Comment by chesik-amd** [comment]
Thanks for the report.

The crash is occurring because there is a barrier reported with a non-zero layout transition count, but there does not appear to be any layout transition instrumentation for that barrier (so RGP ends up dereferencing a null pointer).  We can prevent the crash in RGP, but the data for this barrier will not be accurate unless the driver is updated to include the expected layout transition instrumentation.

**Comment by chesik-amd** [comment]
The crash issue should be resolved in the 1.14.1 patch release that was published today.
